x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/OperationModeUpdateTask.java

```diff
@@ -4,7 +4,7 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-package org.elasticsearch.xpack.ilm;
+package org.elasticsearch.xpack.core.ilm;
```

Comment:
Should this class and `OperationMode` live in `org.elasticsearch.xpack.core`
(as it's used by both ilm and slm)

Could be done in follow-up as well

Reply:
Yeah I struggled with where to put this, but since it depended on several classes that were already in `org.elasticsearch.xpack.core.ilm` I put it there. A follow-up to move all of these into some other package (`org.elasticsearch.xpack.core.lifecycle` maybe?) would make sense.